pyproject.toml

```diff
@@ -94,10 +94,13 @@ dynamic = [
 dependencies = [
   "crate[sqlalchemy]",
   "cratedb-toolkit",
-  'importlib-resources; python_version < "3.9"',
-  "meltanolabs-target-postgres==0.0.9",
+  'importlib-resources; python_version < "3.9"', # "meltanolabs-target-postgres==0.0.9",
```

Comment:
That's apparently a rendering glitch by `pyproject-fmt`.

What I wrote down, to signal that `meltanolabs-target-postgres` was only hooked off temporarily, was:
```
'importlib-resources; python_version < "3.9"',
# "meltanolabs-target-postgres==0.0.9",
```

`pyproject-fmt` was so kind to fold the comment next to the _previous_ line ;]. Is it legit?

/cc @gaborbernat, @edgarrmondragon